lib/ansible/modules/cloud/vmware/vmware_guest_facts.py

```diff
@@ -44,6 +44,11 @@
      description:
      - UUID of the instance to manage if known, this is VMware's unique identifier.
      - This is required if name is not supplied.
+   moid:
+     description:
+     - Manage Object ID of the instance to manage if known, this is an unique identifier only within a single vCenter instance.
```

Comment:
```suggestion
     - Managed Object ID of the instance to manage if known, this is a unique identifier only within a single vCenter instance.
```

Reply:
Thanks again! Had to look up why `a unique` and not `an unique`, learned the rule that a vowel is an but there are some exceptions. You learn every day :)